ddtrace/tracer/tracer_test.go

```diff
@@ -1095,6 +1098,24 @@ func decode(p *payload) (spanLists, error) {
 	return traces, err
 }
 
+func (t *dummyTransport) waitFlush(n int, timeout time.Duration) {
+	a := time.After(timeout)
+	for {
+		select {
+		case <-a:
+			panic("timed out waiting for flush")
+		default:
+			t.Lock()
+			l := len(t.traces)
+			t.Unlock()
+			if l >= n {
```

Comment:
I think we should be exact here, right? We don't want more than what we expect.

Reply:
Expecting the exact amount seems to work.